BEST-PRACTICES.md

```diff
@@ -0,0 +1,49 @@
+  - SUSHI uses [Jest](https://jestjs.io/) as a testing framework.
+  - To run the full test suite, run `npm test`. Ensure all tests are passing.
+- Follow the code style and conventions as enforced by the lint configuration and as evidenced by the existing code.
+  - SUSHI uses [ESLint](https://eslint.org/) for code linting.
+  - To run the linter on all code, run `npm run lint`. Ensure there are no issues reported.
+- Follow the code formatting as enforced by the formatter configuration.
+  - SUSHI uses [Prettier](https://prettier.io/) for code formating.
+  - To run the prettier on all code, run `npm run prettier`. Ensure there are no issues reported.
+- Ensure any new dependencies do not contain any known security vulnerabilities
+  - To check for known security vulnerabilities, we recommend using [npm-audit](https://fshschool.org/). Run `npm audit` and ensure there are no new issues on your branch.
+- Update documentation to reflect any user-facing changes.
+  - Documentation updates may include, but are not limited to, the project [README](README.md) and [FSH School](https://fshschool.org/).
+  - If changes are required to FSH School, follow the instructions for contributing in the [project repository](https://github.com/FSHSchool/site).
+
+## Making a Pull Request
+
+We recommend the following best practices for creating a high quality pull request:
+
+- Review your own PR before marking it as ready for review by others. Ensure the only code changes included are ones relevant to the feature or bugfix and that they follow the coding practices outlined above.
+- Ensure your branch is up to date with master. There are a few ways you can update your branch:
+  - Use the "Update branch" button available once you make your PR. This is the recommended approach if you are not comfortable with merging or rebasing.
+  - [Merge](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) master into your branch.
+  - [Rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your branch on master. We only recommend this approach if you are very comfortable with rebasing.
+- Update the title of the PR to provide a short, descriptive summary of the PR.
+  - Keep the title up to date with any changes made during the review process. The title will be used in the commit message and in the release notes, so it is important that it accurately reflects the current state of the PR.
+- Follow the pull request template to create a detailed PR description.
+  - Include a detailed description of the changes made in the PR.
+  - Include instructions for how to test the PR. You may want to include a link to sample FSH in FSH Online to demonstrate a bug or attach a sample project that highlights new or improved behavior.
+  - [Link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) that the PR addresses.
```

Comment:
Maybe add a bullet to link any relevant Zulip conversations (if applicable)?

Reply:
Hmm I think I like this idea, but one thought I had is -- do we really want the Zulip link in the issue itself? If it is there, do we also need it on the PR? I think you can make an issue template as well. Is it worth creating one of those now?

Reply:
Yeah, as noted above, the issue is a better place for that. And I do agree it would be good to create an issue template. But that's outside the scope of this task, so it's up to you if you want to add it.

Reply:
Rather than trying to get the issue template working as part of this PR/task, I added a sentence to both the Best Practices document and the Contributing document about including a link to relevant Zulip conversations in [a1a2e34](https://github.com/FHIR/sushi/pull/1474/commits/a1a2e34c90adfb63f712189522f8dc5196308d96). I did just refer to it as "Zulip" rather than "chat.fhir.org", but I'm not sure if that's how it is widely referred to. Let me know what you think.